core/tests/test_views.py

```diff
@@ -117,6 +164,61 @@ def test_add_comment_unauthorized(client: Client, post: Post) -> None:
     assert reverse("login") in response.url
 
 
+def test_reported_list_post_by_admin(client: Client, admin: User, post: Post, report_data: dict) -> None:
+    client.force_login(admin)
+    response = client.post(reverse("post-report", kwargs={"pk": post.pk}), data=report_data)
+    assert response.status_code == 302
+    assert reverse("home") in response.url
+    response = client.get(reverse("post-list-reported"))
+    assert response.status_code == 200
+    reports_count = PostReport.objects.filter(verified=False).count()
+    assert reports_count > 0
+
+
+def test_reported_list_post_by_user(client: Client, user: User, post: Post, report_data: dict) -> None:
+    client.force_login(user)
+    response = client.post(reverse("post-report", kwargs={"pk": post.pk}), data=report_data)
+    assert response.status_code == 302
+    response = client.get(reverse("post-list-reported"))
+    assert response.status_code == 302
+    assert reverse("home") in response.url
+
+
+def test_reported_list_post_by_anonymous_user(client: Client) -> None:
+    response = client.get(reverse("post-list-reported"))
+    assert response.status_code == 302
+    assert "/users/login/?next=/core/reported-posts/" in response.url
+
+
+def test_reported_detail_post_by_user(client: Client, user: User, post: Post, report_data: dict) -> None:
+    client.force_login(user)
+    response = client.post(reverse("post-report", kwargs={"pk": post.pk}), data=report_data)
+    assert response.status_code == 302
+    response = client.get(reverse("reported-post", kwargs={"pk": 1}))
```

Comment:
Now wait. This is ID of existing or not exiting post? 
if existing, then comment below. If not existing, then the code should be 404 in the next line.

Reply:
The user does not have access to the application overview.